src/ProcessManagerBundle/Maintenance/QueueTask.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Process Manager.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2018 Jakub Płaskonka (jplaskonka@divante.pl)
```

Comment:
@roynilsson I am not the author of this file :)

Reply:
:D, Copyright is always me, even if someone else created the file... at least it should be me...

Reply:
I guess copyright to you, but @author can be set to @roynilsson ?

Reply:
You're such a great developer @kubaplas that you write code without knowing it! Or maybe I made a copy-paste oops. I will fix that